crates/ironrdp-web/src/session.rs

```diff
@@ -212,13 +214,22 @@ impl iron_remote_desktop::SessionBuilder for SessionBuilder {
         self.clone()
     }
 
+    #[expect(clippy::cast_possible_truncation, clippy::cast_sign_loss)]
```

Comment:
suggestion: There is a function `f64_to_u16_saturating_cast` at the bottom of the file, I suggest you implement something similar to isolate the lint exception further. In your case, that’s not `saturating`, so it would be `f64_to_u16_cast(value: f64) -> Option<u16>`. Alternatively, it’s possible to create a local scope.

```rust
#[expect(clippy::cast_possible_truncation, clippy::cast_sign_loss)]
let limit = { outbound_message_size_limit as u32 };
```